dotnet/samples/Demos/ContentSafety/Exceptions/AttackDetectionException.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace ContentSafety.Exceptions;
+
+public class AttackDetectionException : Exception
+{
+    public AttackDetectionException()
```

Comment:
Can you add some API documentation.

Is there more information about the attack that can be included in this exception?

Reply:
Sure, added a documentation and extended this exception to include more details:
![image](https://github.com/microsoft/semantic-kernel/assets/13853051/aa1ef6e7-c419-4217-ab32-8ce57a0acbc5)